persist/fs/commitlog/types.go

```diff
@@ -49,8 +48,6 @@ const (
 
 // CommitLog provides a synchronized commit log
 type CommitLog interface {
-	io.Closer
```

Comment:
Out of curiosity, this is because we needed to use file mode for mockgen and we don't want to supply aux files?